src/main/resources/jenkins/plugins/openstack/compute/SectionDisabled/config.jelly

```diff
@@ -0,0 +1,24 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials" >
+
+    <f:entry title="${%Enabled}" field="enabledByChoice">
+        <f:checkbox/>
+    </f:entry>
+
+    <f:invisibleEntry>
+        <f:checkbox field="disabledBySystem"/>
```

Comment:
Oh, the necessity to do this suggests *disabled by system* and *disabled by choice* probably should be separate concepts.

Reply:
I am not proud of the Jelly code.

From memory... having the invisible fields allowed the validation code to get called _separately_ from the user-selectable checkbox so that the "this didn't work so we automatically disabled it for a while" text is distinct from the "the user deliberately switched this off" field.

I'm no expert on Jelly and this was what I came up with - if you can improve upon it, please do so (or help @MichalAugustyn do so).